src/templates/components/radio.html

```diff
@@ -1,5 +1,29 @@
 <ng-form name="{{ componentId }}" ng-model="data[component.key]" custom-validator="component.validate.custom">
-  <div ng-class="component.inline ? 'radio-inline' : 'radio'" ng-repeat="v in component.values track by $index">
+  <fieldset ng-if="component.fieldSet">
```

Comment:
Why not just always make this available. Does it affect the UI in any way?

Reply:
What if some users do not want these configurations?